paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -2599,6 +2601,7 @@ class PaymentSheet internal constructor(
     }
 
     @Parcelize
+    @Serializable
```

Comment:
I'm a bit concerned about making this serializable (it's a public model, widely used). Can we create a custom serializer for it / parse it manually?